docs/how-to/h-deploy.md

```diff
@@ -0,0 +1,111 @@
+# How to deploy Charmed Kafka
+
+To deploy a Charmed Kafka cluster on a bare environment, it is required to:
+1. Setup a Juju Controller
+2. Setup a Juju Model
+3. Deploy Charmed Kafka and Charmed ZooKeeper
+4. (Optional) Create an external admin user
+
+In the next subsections we will cover these steps separately by referring to 
+relevant Juju documentation and providing details on the Charmed Kafka specifics.
+If you already have a Juju controller and/or a Juju model, skip the related steps.
+
+## Juju Controller setup
+
+Before deploying Kafka, make sure you have a Juju controller accessible from 
+your local environment using the [Juju client snap](https://snapcraft.io/juju). 
+
+The properties of your current controller can be listed using `juju show-controller`. 
+Make sure that the controller substrate (cloud) is **NOT** K8s. 
+The cloud information can be retrieved with the following command
+
+```commandline
+juju show-controller | yq '.[].details.cloud'
+```
+
+> **IMPORTANT** If the cloud is `k8s`, please refer to the [Charmed Kafka K8s documentation](/t/charmed-kafka-k8s-documentation/10296) instead.
+
+You can find more information on how to bootstrap and configure a controller for different 
+clouds [here](https://juju.is/docs/juju/manage-controllers#heading--bootstrap-a-controller). 
+Make sure you bootstrap a `machine` Juju controller. 
+
+## Juju Model setup
+
+You can create a new Juju model using 
+
+```
+juju add-model <model>
+```
+
+Alternatively, you can use a pre-existing Juju model and switch to it by 
+
+```
+juju switch <model-name>
+```
+
+Make sure that the model is **not** a `k8s` type. The type of the model 
+can be obtained by 
+
+```
+juju show-model | yq '.[].type'
+```
+
+> **IMPORTANT** If the model is `k8s`, please refer to the [Charmed Kafka K8s documentation](https://discourse.charmhub.io/t/charmed-kafka-k8s-documentation/10296) instead.
+
+
+## Deploy Charmed Kafka and Charmed ZooKeeper
+
+The Kafka and ZooKeeper charms can both be deployed as follows:
+
+```shell
+$ juju deploy zookeeper --channel 3/edge -n <zookeeper-units>
+$ juju deploy kafka --channel 3/edge -n <kafka-units>
+```
+
+After this, it is necessary to connect them:
```

Comment:
```suggestion
After this, it is necessary to [integrate](https://juju.is/docs/juju/relation) them:
```

Reply:
yes, I'm afraid we may have a bit of "relate" all over the place right now (even in other docs that are not included in this PR). I'll try to do a search and replace, but in one commit to hopefully address this consistently